.github/workflows/kotlin.yml

```diff
@@ -0,0 +1,24 @@
+name: kotlin
+
+concurrency:
```

Comment:
So if this check fails does it cancel all other checks?

Reply:
No, it just means that if you push an update to the same ref, and there's already a run in progress for that ref, that run will be canceled. This is to avoid having multiple CI runs, where most of them would test the old state of the ref.

For example: 
```
push commit X to branch foo
<github starts a new run (R1) for brach foo, based on state X>
push commit Y to branch foo
<github sees the above and that run R1 is still active so it cancel R1 and starts a new run (R2) based on state Y
```
```